openlibrary/templates/account/mybooks.html

```diff
@@ -32,8 +32,10 @@
   </div>
   $if current_goal:
     <span id="reading-goal-container">
-      $:render_template('check_ins/reading_goal_progress', [current_goal])
+      $:render_template('check_ins/reading_goal_progress', **{"goals":[current_goal], "count": mybooks["already-read"].total_results})
```

Comment:
I think other commits may have made their way into your PR.

I am wary of making a suggestion that may make things worse, but it may be worth looking at the documentation for `git rebase`, with emphasis on `git rebase -i`. I *think* that should allow you `pick` the specific commits you want, which should allow you to not use this commit in your PR.